docs/MO_DG/prepare_model/convert_model/Convert_Model_From_PyTorch.md

```diff
@@ -0,0 +1,45 @@
+# Converting a PyTorch* Model {#openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_PyTorch}
+
+PyTorch* framework is supported through export to ONNX\* format. A summary of the steps for optimizing and deploying a model that was trained with the PyTorch\* framework:
+
+1. [Configure the Model Optimizer](../Config_Model_Optimizer.md) for ONNX\*.
+2. [Export PyTorch model to ONNX\*](#export-to-onnx) if your model is not already frozen or skip this step and use the [instruction](#loading-nonfrozen-models) to a convert a non-frozen model.
```

Comment:
I think it should be in the first step because the MO tool is not needed for the export

Reply:
Also the description of this step seems a bit messy. Please correct it.